var/spack/repos/builtin/packages/comgr/package.py

```diff
@@ -157,6 +160,9 @@ class Comgr(CMakePackage):
             "rocm-device-libs@" + ver, when="@{0} ^llvm-amdgpu ~rocm-device-libs".format(ver)
         )
 
+    for ver in ["5.5.0", "5.5.1"]:
+        depends_on("rocm-core@" + ver, when="@" + ver)
```

Comment:
In the deb/rpm packages, the declared dependencies on rocm-core exist only so that you can `apt remove rocm-core` or `dnf remove rocm-core` and the package manager will remove all other rocm components.

The new thing in ROCm 5.5 is that a rocmcore library was added to the rocm-core package. The library that can be used to query the version number of the rocm release. However, nothing should actually be using that library. The version APIs it provides only really make sense in the context of the AMD deb/rpm releases.